package.json

```diff
@@ -46,5 +46,8 @@
     "cross-env": "^7.0.3",
     "patch-package": "^6.4.7",
     "style-dictionary": "3.1.1"
+  },
+  "dependencies": {
```

Comment:
This is the first time we've listed a dependency in this root workspace `package.json`. What's rationale/impact of doing so? Does this mean all projects need to be on 4.6.4? That doesn't seem to be the case, since nimble-tokens is still on 4.3.2. If it's just needed for building, should it be a `devDependency` instead? Or if it was a change made by the Angular CLI, could we try reverting it and see what breaks?

Reply:
Whoops, this should be a dev dependency. This is required to cap the all the workspaces at 4.6 because of fast-foundation issue [6365](https://github.com/microsoft/fast/issues/6365). There may also have been an incompatibility with storybook. I don't remember exactly. nimble-components and tokens remain at 4.3. I didn't consider it my decision to change those.